class.jetpack-debugger.php

```diff
@@ -213,7 +213,7 @@ public static function jetpack_debug_display_handler() {
 
 					<input type="hidden" name="contact_form" id="contact_form" value="1">
 					<input type="hidden" name="blog_url" id="blog_url" value="<?php echo esc_attr( site_url() ); ?>">
-					<input type="hidden" name="subject" id="subject" value="from: <?php echo esc_attr( site_url() ); ?> Jetpack contact form">
+					<input type="hidden" name="subject" id="subject" value="<?php echo ( Jetpack::is_development_version() ? 'BETA ' : '' ); ?> from: <?php echo esc_attr( site_url() ); ?> Jetpack contact form">
```

Comment:
I guess we could start translating the subject line, it would make sense when we reply to emails sent by non-english speakers. I'll update my PR. :+1: